Packages/com.unity.inputsystem/InputSystem/Editor/AssetImporter/InputActionImporter.cs

```diff
@@ -58,7 +50,7 @@ private static InputActionAsset CreateFromJson(AssetImportContext context)
             string content;
             try
             {
-                content = EditorHelpers.ReadAllText(context.assetPath);
```

Comment:
Some of this extra refactoring seemed unnecessary for this pr - tidy maybe but had made the pr longer to review